modules/core/core-elasticsearch-server/src/main/java/com/enonic/xp/elasticsearch/server/config/ElasticsearchServerConfigResolver.java

```diff
@@ -49,13 +49,14 @@ private String getPathOfDirAsString( final String value, final String defaultSub
             return value;
         }
 
-        return HomeDir.get().toFile().toPath().resolve( defaultSubPath ).toAbsolutePath().toString();
-    }
-
-    public String resolveElasticServerDir()
-    {
-        final String value = configuration.esServerDir();
-        return getPathOfDirAsString( value, "../elasticsearch" );
+        try
+        {
+            return Files.createDirectories( HomeDir.get().toFile().toPath().resolve( defaultSubPath ) ).toAbsolutePath().toString();
```

Comment:
why toFile().toPath() ?

Reply:
HomeDir.get() returns an instance of HomeDir, so we need to call toFole and toPath